src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const refToState = state;
```

Comment:
Why do you need it? Can't you just use state?

Reply:
I can, it was late at night:)